src/layer/base-layer.ts

```diff
@@ -30,7 +50,7 @@ export interface ILayerSettings {
   depth?: number | null;
   weights?: KernelOutput | null;
   deltas?: KernelOutput | null;
-  name?: string | null;
+  title?: string | null;
```

Comment:
We are dealing here with both `undefined and null`, not a big deal but I think `?` should be preferred.

Reply:
nevermind, we can update it latter if required.